### PR TITLE
Remove param and green up csv_importer_spec

### PR DESCRIPTION
**Pull request by bmquinn**
- Removes @model parameter from Importer::CSVImporter#initialize
- Refactors Importer::CSVImporter spec to use aws-sdk functionality

### CODE REVIEW

File: spec/lib/importer/csv_importer_spec.rb [hunk 1]
```diff
@@ -1,39 +1,23 @@
 require 'rails_helper'
 require 'importer'
+require 'aws-sdk'
```
Comment (csyversen) [inline spec/lib/importer/csv_importer_spec.rb]: i'm getting an error with this line: ```
An error occurred while loading ./spec/lib/importer/csv_importer_spec.rb.
Failure/Error: require 'aws-sdk'

LoadError:
  cannot load such file -- aws-sdk
# ./spec/lib/importer/csv_importer_spec.rb:3:in <top (required)>'
No examples found.

Randomized with seed 40214


Finished in 0.13636 seconds (files took 8.31 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

i think we might need to mess with our `Gemfile` groups a bit, and maybe we only want to require the aws-sdk module that we're using in this spec (although i don't care a ton about that right now)

Comment (bmquinn) [reply spec/lib/importer/csv_importer_spec.rb]: Can you try `bundle install --with=aws` and see if that gets past the error?

Comment (csyversen) [reply spec/lib/importer/csv_importer_spec.rb]: cool, doing that and setting up minio with the updated instructions worked

File: spec/lib/importer/csv_importer_spec.rb [hunk 2]
```diff
@@ -1,39 +1,23 @@
...
     let(:image_factory) { double }
 
     it 'creates new images and collections' do
-      expect(Importer::Factory::CollectionFactory).to receive(:new)
-        .with(hash_excluding(:type), image_directory)
-        .and_return(collection_factory)
-      expect(collection_factory).to receive(:run)
-      expect(Importer::Factory::ImageFactory).to receive(:new)
-        .with(hash_excluding(:type), image_directory)
+      expect(Importer::Factory::ImageFactory).to have_received(:new)
```
Comment (csyversen) [inline spec/lib/importer/csv_importer_spec.rb]: this test is now weirdly giving me an error: 
```
Failures:

  1) Importer::CSVImporter when the model specified on the row creates new images and collections
     Failure/Error:
       expect(Importer::Factory::ImageFactory).to have_received(:new)
         .with(hash_excluding(:type), csv_resource)
         .and_return(collection_factory)

     NoMethodError:
       undefined method `and_return' for #<RSpec::Mocks::Matchers::HaveReceived:0x00007faaa54e53e8>
     # ./spec/lib/importer/csv_importer_spec.rb:19:in `block (3 levels) in <top (required)>'

Finished in 0.75782 seconds (files took 11.28 seconds to load)
1 example, 1 failure
```

is that working on your machine?

**Comment by csyversen** [comment]
tests pass now! @mbklein 's pr will have changes that allow everything to work, but for right now we need to create a `config/settings.local.yml` with this entry ```
aws:
  endpoint: http://localhost:9000
```

**Pull request merged**